sympy/utilities/tests/test_lambdify.py

```diff
@@ -1130,6 +1130,22 @@ def test_issue_15827():
     if not numpy:
         skip("numpy not installed")
     A = MatrixSymbol("A", 3, 3)
-    f = lambdify(A, 2*A)
+    B = MatrixSymbol("B", 3, 2)
+    C = MatrixSymbol("C", 2, 3)
```

Comment:
I would use shapes that are only valid in one order. Like (2, 3), (3, 4), (4, 5). Also it would be good to test a product of at least three matrices.